cypress/integration/feedback.spec.js

```diff
@@ -0,0 +1,80 @@
+/* global describe, it, beforeEach, cy, expect */
+/* eslint-disable-next-line */
+import React from 'react'
+import fakeData from '../../js/data/report.json'
+import sampleRubric from '../../public/sample-rubric'
+
+describe('Provide Feedback', function () {
+  beforeEach(() => {
+    // cypress currently doesn't handle window.fetch so:
+    // diable browser's fetch this way the fetch polyfill is used which uses XHR
```

Comment:
Looks like this typo snuck back in the last commit :P 

```suggestion
    // disable browser's fetch this way the fetch polyfill is used which uses XHR
```